### PR TITLE
Iht 4275 Gifts totals in pre-submission PDF

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/iht/utils/pdf/XmlFoToPDF.scala [hunk 1]
```diff
@@ -107,6 +107,7 @@ trait XmlFoToPDF {
 
   private def createPreSubmissionTransformer(registrationDetails: RegistrationDetails,
                                              applicationDetails: ApplicationDetails): Transformer = {
+    def getOrMinus1(value:Option[BigDecimal]):BigDecimal = value.fold(BigDecimal(-1))(identity)
```
Comment (dmbeer) [inline app/iht/utils/pdf/XmlFoToPDF.scala]: Better to use CommonHelper.getOrZero

File: app/iht/utils/pdf/XmlFoToPDF.scala [hunk 2]
```diff
@@ -120,9 +121,10 @@ trait XmlFoToPDF {
     val dateOfPredeceased = applicationDetails.widowCheck.flatMap { x => x.dateOfPreDeceased }
 
     setupCommonTransformerParametersPreAndPost(transformer, registrationDetails, preDeceasedName, dateOfMarriage,
-      applicationDetails.totalAssetsValue, applicationDetails.totalLiabilitiesValue,
-      applicationDetails.totalExemptionsValue, applicationDetails.totalGiftsValue)
+      applicationDetails.totalAssetsValue, applicationDetails.totalLiabilitiesValue, applicationDetails.totalGiftsValue)
 
+    transformer.setParameter("giftsTotalExclExemptions", getOrMinus1(applicationDetails.totalPastYearsGiftsValueExcludingExemptionsOption))
+    transformer.setParameter("exemptionsTotal", getOrMinus1(applicationDetails.totalPastYearsGiftsExemptionsOption))
```
Comment (dmbeer) [inline app/iht/utils/pdf/XmlFoToPDF.scala]: This used for the Exemptions Value as shown on the estate overview. Not Exemptions for Gifts

File: conf/pdf/templates/iht-component-templates.xsl [hunk 3]
```diff
@@ -90,6 +90,65 @@
         </fo:table-row>
     </xsl:template>
 
+
```
Comment (dmbeer) [inline conf/pdf/templates/iht-component-templates.xsl]: Extra Line Break

File: conf/pdf/templates/iht-component-templates.xsl [hunk 4]
```diff
@@ -90,6 +90,65 @@
...
+                        </xsl:when>
+                        <xsl:otherwise>
+                            &#xA3;<xsl:value-of select="$value3"/>
+                        </xsl:otherwise>
+                    </xsl:choose>
+                </fo:block>
+            </fo:table-cell>
+        </fo:table-row>
+    </xsl:template>
+
```
Comment (dmbeer) [inline conf/pdf/templates/iht-component-templates.xsl]: Extra Line Breaks

File: app/iht/utils/pdf/XmlFoToPDF.scala (hunk 1 above)
Comment (grantrobson) [inline app/iht/utils/pdf/XmlFoToPDF.scala]: Can't use this as it's possible for the user to enter zero as a value.

File: app/iht/utils/pdf/XmlFoToPDF.scala (hunk 2 above)
Comment (grantrobson) [inline app/iht/utils/pdf/XmlFoToPDF.scala]: Done

File: conf/pdf/templates/iht-component-templates.xsl (hunk 3 above)
Comment (grantrobson) [inline conf/pdf/templates/iht-component-templates.xsl]: Done

File: conf/pdf/templates/iht-component-templates.xsl (hunk 4 above)
Comment (grantrobson) [inline conf/pdf/templates/iht-component-templates.xsl]: Done

**Pull request merged**